### PR TITLE
password cracked

### PR DESCRIPTION
**Pull request by GuYOAli**
1. Password Cracker
2. Bug fixed
3. No more error

### CODE REVIEW

File: password.txt [hunk 1]
```diff
@@ -1,2 +1 @@
-forget password
-reset my password
+your password is cracked
```
Comment (GuyoIbra) [inline password.txt]: It looks good but it would be better if you capitalise first letters.

**Review by GuyoIbra — CHANGES_REQUESTED** [review]
make another commit and fix the letters

File: password.txt (hunk 1 above)
Comment (GuyoIbra) [reply password.txt]: Okay SIr Thank you for your response

Comment (GuYOAli) [reply password.txt]: okay Sir

File: password.txt [hunk 2]
```diff
@@ -1 +1 @@
-your password is cracked
+Your Password Cracked
```
Comment (GuyoIbra) [inline password.txt:1]: Perfect now we can merge.

**Review by GuyoIbra — APPROVED** [review]
Ready to merge.

**Pull request merged**